Card.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>replit</title>
```

Comment:
the title is usually related to the content of the webpage. Ignore this if its related to the content.